samples/client/petstore/java/jersey2-java8/src/main/java/org/openapitools/client/ApiClient.java

```diff
@@ -787,7 +794,7 @@ public String escapeString(String str) {
    * @return String
    * @throws ApiException API exception
    */
-  public String serializeToString(Object obj, Map<String, Object> formParams, String contentType) throws ApiException {
+  public String serializeToString(BodyHolder obj, Map<String, Object> formParams, String contentType) throws ApiException {
```

Comment:
@wing328 , I'm a bit confused why `updateParamsForAuth` invokes serializeToString, but the request body is serialized using a different function. That means an authorization scheme that processes the message body may provide incorrect authorization data.
For example, the message digest may be different.

Reply:
I agree ideally they should be using the same function to come up with the request body.

Reply:
thanks. I propose that we tackle this in a follow-up PR, what do you think? I see some discrepancies between serializeToString and serialize.